.gitlab-ci.yml

```diff
@@ -102,7 +103,7 @@ before_script:
       when: on_success
 # ================== templates ================== #
 .base_template: &base_template
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/websites-images:webops-site-build-15527163
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/websites-images:webops-site-build-devin.ford_slack-url-update
```

Comment:
This will be updated to the normal naming convention with a new number at the end once the pipeline runs